api/storage_service.cc

```diff
@@ -74,6 +74,21 @@ sstring validate_keyspace(http_context& ctx, const parameters& param) {
     return validate_keyspace(ctx, param["keyspace"]);
 }
 
+struct keyspace_and_table_name {
+    sstring keyspace;
+    sstring table;
+};
+
+keyspace_and_table_name validate_table(http_context& ctx, sstring ks_name, sstring table_name) {
```

Comment:
IMHO, it does not make sense to introduce this type for boxing and unboxing the names. we will pass them separately down to `describe_ring_as_json_for_table()` anyway, instead as a single struct.

Reply:
might want to mark this helper with `static`, as it does not need external linkage.

Reply:
I think we can make validate_table void.

Reply:
and static too.